templates/_macros/quote-wrapper.jinja

```diff
@@ -0,0 +1,137 @@
+  - citation_name (string) (optional): The name of the source being quoted
+  - citation_title (string) (optional): The title of the source being quoted
+  Slots
+  - logo (optional): The logo of the source being quoted
+  - heading_cta (optional): The call to action block to be displayed next to the heading
+  - cta (optional): Contents of the call to action block to be displayed below the quote
+  - image (optional): An image to be displayed below the quote
+#}
+{%- macro quote_wrapper(
+  heading_text="",
+  quote_heading_level=2,
+  quote_text="",
+  citation_name="",
+  citation_title=""
+) -%}
+  {% set heading_cta_content = caller('heading_cta') %}
+  {% set has_heading_cta =  heading_cta_content|trim|length > 0 %}
+  {% set has_heading_text = heading_text|length > 0 %}
+  {% set has_heading_row = has_heading_text or has_heading_cta %}
+  {% set logo_content = caller('logo') %}
+  {% set has_logo = logo_content|trim|length > 0 %}
+  {% set has_citation = citation_name|trim|length > 0 %}
+  {% set has_citation_title = has_citation and citation_title|trim|length > 0 %}
+  {% set cta_content = caller('cta') %}
+  {% set has_cta = cta_content|trim|length > 0 %}
+  {% set image_content = caller('image') %}
+  {% set has_image = image_content|trim|length > 0 %}
+
+  {# Validate heading level selection #}
+  {% if quote_heading_level not in [2, 4] %}
+    {% set quote_heading_level = 2 %}
+  {% endif %}
+
+  {%- macro _quote_body() -%}
+    <div class="p-section--shallow">
+      <blockquote class="p-pull-quote u-no-margin--top u-no-padding--left">
+        <h{{ quote_heading_level }} class="u-no-padding--top">
+          <i>
+            “{{ quote_text }}”
+          </i>
+        </h{{ quote_heading_level }}>
+      </blockquote>
+    </div>
+  {%- endmacro %}
+
+  {%- macro _citation_block() -%}
+    {%- if has_citation -%}
+      <!--Optional citation block-->
+      <div class="p-section--shallow">
+        <h5 class="u-no-margin--bottom u-no-padding--top">{{ citation_name }}</h5>
+        {%- if has_citation_title %}
+          <!--Optional citation source title-->
+          <p class="u-text--muted u-no-padding--top">{{ citation_title }}</p>
+        {%- endif -%}
+      </div>
+    {%- endif -%}
+  {%- endmacro %}
+
+  {% if has_heading_row %}
+    <!--Optional heading -->
+    <div class="row">
+      <hr class="p-rule--highlight">
+      {%- if has_heading_text %}
+        <!--Optional heading text-->
+        <div class="col-3 col-medium-3">
+          <p class="p-muted-heading">{{ heading_text }}</p>
+        </div>
+      {%- endif -%}
+
+      {%- if has_heading_cta %}
+        <!--Optional heading CTA -->
+        <div class="col-3 col-medium-3 col-start-large-10 col-start-medium-5">
+          {#- TODO need a borderless variant of CTA block otherwise there is a top border here -#}
+          <div class="p-cta-block" style="border: none">
+           {{ heading_cta_content }}
+          </div>
```

Comment:
We need to decide to either merge and use borderless CTA (from #5278) or just pass in link text and href and render a simple link here, without caring about text wrapping

Reply:
Adjusted this to simply use a slot for passing in link content